index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./src/styles.css">
+    <title>Document</title>
+    <h1>About Me</h1>
+</head>
+<body>
+        <P>Interested in merging poetry and photography</P>
+       <img src=https://images.unsplash.com/photo-1474204075013-fafcfee9bfd7?ixlib=rb-1.2.1&auto=format&fit=crop&w=2125&q=80>
```

Comment:
You need an alt="description of the image" inside this img tag for accessibility.